sanic/static.py

```diff
@@ -93,6 +98,17 @@ def register(app, uri, file_or_directory, pattern,
                     headers=headers,
                     content_type=guess_type(file_path)[0] or 'text/plain')
             else:
+                if stream_large_files:
```

Comment:
Can you provide a benchmark before/after this change, particularly when this feature is disabled? `wrk -t 12 -c 100 -d 20 localhost:8000/some_file.txt` should be sufficient.